force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -83,7 +113,14 @@ public with sharing class FlowControlLoopsCollections {
      * @return A string indicating whether the number is "Even" or "Odd", or null if the number is null.
      */
     public static String checkEvenOdd(Integer a) {
-        return null; // Replace null with the variable you used to store the result
+        if(a == null){
+            return null;
+        }
+        Integer result = System.Math.mod(a,2);
+        if (result == 0) {
+            return (String)('Even');
+        } else {return (String)('Odd');}
```

Comment:
Apex usually associates characters that are in single quotes as strings. Try to see if you can do it without casting it into a string